org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintUiPlugin.java

```diff
@@ -80,11 +76,9 @@ public class SonarLintUiPlugin extends AbstractUIPlugin {
   @Nullable
   private SonarLintConsole console;
 
-  private ListenerFactory listenerFactory;
-
   private static final WindowOpenCloseListener WINDOW_OPEN_CLOSE_LISTENER = new WindowOpenCloseListener();
   private static final SonarLintPostBuildListener SONARLINT_POST_BUILD_LISTENER = new SonarLintPostBuildListener();
-  private static final SonarLintProjectEventListener SONARLINT_PROJECT_EVENT_LISTENER = new SonarLintProjectEventListener();
+  private static final SonarLintVcsCleanupListener SONARLINT_PROJECT_EVENT_LISTENER = new SonarLintVcsCleanupListener();
```

Comment:
The static variable should be renamed too. And after more thinking, a better name is maybe:
```suggestion
  private static final SonarLintVcsCacheCleaner SONARLINT_VCS_CACHE_CLEANER = new SonarLintVcsCacheCleaner();
```

Reply:
Done